### ISSUE TITLE
strict=False does not work for referenced files

### ISSUE THREAD

**Issue by sam-hernandez-smartshoot** [jfinkhaeuser/prance#83]
## Expected Behaviour
`strict=False` should work with ResolvingParser when operations are defined in a separate file.
```
$ prance validate --no-strict bad_spec.yaml
Processing "bad_spec.yaml"...
 -> Resolving external references.
Validates OK as OpenAPI 3.0.3!
```

### Minimal Example Spec

`bad_spec.yaml`
```
openapi: 3.0.3

info:
  title: OpenAPI spec to demonstrate stringify reference issue
  description: |
    This file is references one GET operation.
    and should validate fine with prance using `strict=False` but currently breaks.
  version: '1.0'

paths:
  /bad:
    $ref: 'bad_get.yaml'
```

`bad_get.yaml`
```
get:
  responses:
    200:
      description: Success
```

## Actual Behaviour

Validation fails:
```
$ prance validate --no-strict bad_spec.yaml
Processing "bad_spec.yaml"...
 -> Resolving external references.
ERROR in "bad_spec.yaml" [ValidationError]: expected string or bytes-like object -- Strict mode enabled (the default), so this could be due to an integer key, such as an HTTP status code.
```

## Steps to Reproduce
Create the files above (`bad_spec.yaml` and `bad_get.yaml`) and then run:
```
prance validate --no-strict bad_spec.yaml
```

Note that this works fine:
```
$ openapi-spec-validator bad_spec.yaml
OK
```

## Environment
- OS: Mac OS X 10.14.6
- Python version: 3.7.3
- Swagger/OpenAPI version used: OpenAPI 3.0.3
- Backend: openapi-spec-validator

@jfinkhaeuser

**Comment by jfinkhaeuser** [comment]
Thanks! I'll take a look.

**Comment by jfinkhaeuser** [comment]
Fixed in v0.20.2

**Comment by jayvdb** [comment]
@jfinkhaeuser , 0.20.2 hasnt been released to PyPI yet.
